src/main/java/com/google/sps/models/StandardCluster.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.servlets;
+import java.util.ArrayList;
+
+public class StandardCluster {
+    String title = "Rock out while you work out";
```

Comment:
We probably want to pass in a title. I dont think all the clusters have the same title. You can do this by adding a constructor.

```
public StandardCluster(String title) {
   this.title = title;
}
```